tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature

```diff
@@ -2886,4 +2886,75 @@ Feature: Create a share link for a resource
       | createOnly       | view                 |
       | createOnly       | edit                 |
       | createOnly       | upload               |
-      | blocksDownload   | blocksDownload       |
\ No newline at end of file
+      | blocksDownload   | blocksDownload       |
+
+
+  Scenario Outline: update link share of a file inside a project-space
```

Comment:
```suggestion
  Scenario Outline: update link share of a file inside a project-space (permissions endpoint)
```

Reply:
@saw-jan  should I change it to : 
update link share of a file inside a project-space using permissions endpoint 
OR
update link share of a file inside a project-space (permissions endpoint)